doc/cookbook/CMakeLists.txt

```diff
@@ -15,6 +15,23 @@ configure_file(
     "${BINARY_BUILD_DIR}/conf.py"
     @ONLY)
 
+add_custom_target(cookbook_copy_source
+    ${CMAKE_COMMAND} -E copy_directory
+    "${CMAKE_CURRENT_SOURCE_DIR}/source"
+    "${BINARY_BUILD_DIR}"
+    COMMENT "Copying cookbook Sphinx source to ${BINARY_BUILD_DIR}")
```

Comment:
Could you remove the path names in the comment since cmake output gets obscured by that